### PR TITLE
prow/pr_status: on enter, make the search instead of new line

### LINKED ISSUE #7245: prow: deck: pr: hitting enter in the search query box should run it, not add a new line
When I type in my query and hit enter I was hoping it would run the query, not add a newline to the text in the entry box.

/kind bug
/area prow
/cc @BenTheElder 
/assign @qhuynh96

### PR DESCRIPTION
**Pull request by qhuynh96**
* Fixes #7245

### CODE REVIEW

File: prow/cmd/deck/static/pr-script.js [hunk 1]
```diff
@@ -146,9 +146,17 @@ function createSearchCard() {
     input.rows = 1;
     input.spellcheck = false;
     input.addEventListener("keydown", (event) => {
-        const el = event.target;
-        el.style.height  = "auto";
-        el.style.height = event.target.scrollHeight + "px";
+        if (event.keyCode === 13) {
+            event.preventDefault();
+
+            const query = getPRQuery(input.value);
```
Comment (BenTheElder) [inline prow/cmd/deck/static/pr-script.js]: we should probably refactor this out into a method for submitting the query

Comment (qhuynh96) [reply prow/cmd/deck/static/pr-script.js]: done

**Comment by BenTheElder** [comment]
/area prow
Thanks!

**Pull request merged**